packages/studio/src/store/slices/pages/createPageSlice.ts

```diff
@@ -158,6 +163,29 @@ export const createPageSlice: SliceCreator<PageSlice> = (set, get) => {
         (component) => component.uuid === activeComponentUUID
       );
     },
+    getFirstSelectedComponentUUID: () => {
+      const { selectedComponentUUIDs, getActivePageState } = get();
+      const activePageState = getActivePageState();
+      const componentTree = activePageState?.componentTree;
+      if (!componentTree) {
+        throw new Error(
+          `Error rendering components: component tree is undefined.`
+        );
+      }
+      const orderedSelectedComponentUUIDs =
+        ComponentTreeHelpers.mapComponentTreeParentsFirst(
+          componentTree,
+          (c) => {
+            if (selectedComponentUUIDs.has(c.uuid)) {
+              return c.uuid;
+            }
+            return undefined;
+          }
+        ).filter((uuid) => !!uuid);
+      return orderedSelectedComponentUUIDs.length !== 0
+        ? orderedSelectedComponentUUIDs[0]
```

Comment:
instead of the ternary, can we directly return `orderedSelectedComponentUUIDs[0]` here since it should already be undefined if it doesn't exist?

Reply:
Done